build-system/tasks/presubmit-checks.js

```diff
@@ -596,6 +596,10 @@ const forbiddenTerms = {
       'extensions/amp-access/0.1/iframe-api/access-controller.js',
     ],
   },
+  '(?<!return) allowConsoleError\\(\\(\\) => \\{[\\n\\r\\s]*return': {
```

Comment:
Lookbehinds are brand new, impossible to polyfill.

Reply:
Why is this a problem? This is just a presubmit check running on Travis.

Reply:
Only available on Node 9, and needs a flag on Node 8. http://node.green/#ES2018-features--RegExp-Lookbehind-Assertions